### ISSUE TITLE
Clarify Site Map and Information Architecture

### ISSUE THREAD

**Issue by ALRubinger** [TBD54566975/docs-sitemap#2]
* Sitemap of developer.tbd.website
* Including deep dive into defining the content types in the Docs section:
  * Guides/Docs?
  * Sandbox/Playground?
  * API Docs
  * Tutorials
  * Sample Apps
  * ...all commonly understood by team, with templates, examples for each.
* And decompose into smaller tasks.

Drives the wireframing completion of the Phase 2 of the Site and Docs Section

**Comment by ALRubinger** [comment]
@bobbilee19 has created a start to this in repo: https://github.com/TBD54566975/docs-sitemap

**Comment by bobbilee19** [comment]
The main place I want feedback and discussions to happen on our main developer docs sitemap/sidebar are in this [HackMD file](https://hackmd.io/@mfCHHFWDRcaHJX3OqGUxLQ/sitemap-general). I mentioned this in Discord but definitely got buried so no reviews have happened yet.

@ALRubinger @mistermoe @decentralgabe @dayhaysoos @nitro-neal @kirahsapong @dcrousso @shamilovtim can you look over: add thoughts, additional files, renaming suggestions, reordering suggestions etc!

**Comment by ALRubinger** [comment]
Excellent - I def missed that link in Discord so thanks for flagging! I'll slate that review for tomorrow!

**Comment by ALRubinger** [comment]
I think this is still the current collab working location: https://hackmd.io/RZ5B8Hp6TIGcC9tnDtZ3fw, @bobbilee19?

And here's a link to some of the notes from our sync session this week, especially the content bits @csuwildcat dropped in for Web5 content: https://hackmd.io/5H3-e-0kRg2zL0qhsJakyQ

**Comment by bobbilee19** [comment]
This is getting really close, @kirahsapong added in ideas for SSI and I'm buttoning up web5 js sidebar order.

I would like to chat with a few folks on web5 js sidebar, deepen some of the notes we had on this.

**Comment by bobbilee19** [comment]
Ok the sitemap has been updated a bit in the last few days. Now @kirahsapong is going to take it from me in a way to help get eyes on it from OSE, specifically for the Web5 JS sections. I'll have Kirah update you all more on this tomorrow.

**Comment by dayhaysoos** [comment]
Hey @bobbilee19, just wanted to share some thoughts on sidebar content.

A lot of sidebars in docs sites normally have only one or two words when it comes to content within the sidebar. This makes it easier to style and makes navigation a little more easier and effective.

Right now, we have tutorials like, "Manually Issue a Verifiable Credential". This is a bit wordy and doesn't look as nice on side bars.

I've been looking at examples of how tutorials are set up in other doc sites:

![Screenshot 2023-03-22 at 2 44 43 PM](https://user-images.githubusercontent.com/1852675/227006133-3b5a57a3-d69f-44fe-a27a-0e7484a61773.png)

With docusaurus, the topic is "Internationalization" and within that is Tutorial. What if we had something similar to this with SSI Service?

The structure could look like:

SSI Service > VC > Tutorial

Definitely need to work on how that would look, but either way, I like the idea of associating a "Tutorial" section with each aspect of Web5 vs having a Tutorials section with a bunch of different tutorials that pertain to an aspect of Web5.

Let me know what you think!

**Comment by bobbilee19** [comment]
Yes totally, if you see my repo it shows the tutorials in this way, annnd also I wasn't expecting all tutorials to be listed on the side bar, i'm thinking we do VC > Tutorials and then that opens to an index / listed page of tutorials.

**Comment by bobbilee19** [comment]
Main update here, after standup yesterday the sitemap is waiting on review (kirah is helping out a bit here to get more eyes on it) and will be finalized in Austin with team in breakout session.

**Comment by ALRubinger** [comment]
> Does the copy need to be done and final site map by austin?

I don't think so! My feeling is "done enough to seed the discussion" with folks would be great. You'll have 'em all gathered in one space with their attention so you can get feedback and make decisions there.

**Comment by bobbilee19** [comment]
Hey I'm going to close this for now, it's at it's best before Austin...this is done and will always be evolving in motion this whole week.
